communication/icons/README.md

```diff
@@ -0,0 +1,54 @@
+# Kubernetes Icons Set
+
+These icons are a way to standardize Kubernetes architecture diagrams for presentation. Having uniform architecture diagrams improve understandibility.
+
+## Generate png icons from svg
+
+```shell
+./tools/rasterize.sh
+```
+
+This script will convert all svg into png. It's based on a docker container defined [here](hack/svgconvertor/Dockefile)
+
+## How to use these icons
+
+Each icons can be found in differents formats
+* [png](png)
+* [svg](svg)
+
+There is 2 types of icons
+
+| Kind  | Type       | Icon                             |
+|-------|------------|----------------------------------|
+|  Pod  | Labeled    | ![](./doc/pod-labeled.png)       |
+|  Pod  | Unlabeled  | ![](./doc/pod-unlabeled.png)     |
+
+
+This repository contains slidedeck with all icons and some diagrams
+
+| Link  | Extension  |
+|-------|------------|
+|  [Kubernetes_Icons_Keynote](./slidedeck/Kubernetes_Icons_Keynote.key)  | Apple keynote 09    |
+|  [Kubernetes_Icons_PPTX](./slidedeck/Kubernetes_Icons_PPTX.pptx)  | Microsoft PowerPoint 16    |
+|  [Kubernetes_Icons_GSlide](https://docs.google.com/presentation/d/1lJ0RHWzEKzd6PFePTpukNqPoTIPrvg25_-VS6bmEHIw/edit?usp=sharing)  | Google Slide    |
```

Comment:
This has not been updated with new icons, nor the PDF.  I can't open the others. :)

Do we need these?

Reply:
I don't know, are we considering that pngs are enough ? WDYT ?

However, README is important and I will update it with new icons.